libs/arcade-core/arcade_core/api_wrapper/schema.py

```diff
@@ -0,0 +1,129 @@
+from typing import Literal
+
+from pydantic import BaseModel, Field
+
+from arcade_core.schema import InputParameter, ToolDefinition, ToolInput
+
+
+class ObjectMetadata(BaseModel):
+    """Object metadata (part of the serialized JSON stored in a Wrapper toolkit package)."""
+
+    object_type: str
+    """The type of the object."""
+
+    version: str
+    """The version of the object."""
```

Comment:
This can likely be validated since it is a string.  Semver and numbers only?

Reply:
Good idea, I just committed a field validator using `packaging.Version`